R/tm_data_table.R

```diff
@@ -29,6 +29,14 @@
 #'
 #' @inherit shared_params return
 #'
+#' @section Decorating `tm_data_table`:
+#'
+#' This module generates the following objects, which can be modified in place using decorators:
+#' - `table` (`DT::datatable`)
```

Comment:
(optional): what do you think about using a link?

```suggestion
#' - `table` ([DT::datatable()])
```